x-pack/plugin/core/src/test/java/org/elasticsearch/xpack/core/ilm/CloseFollowerIndexStepTests.java

```diff
@@ -110,6 +110,30 @@ public void onFailure(Exception e) {
         Mockito.verifyNoMoreInteractions(indicesClient);
     }
 
+    public void testCloseFollowerIndexIsNoopForAlreadyClosedIndex() {
```

Comment:
This seems related to the open/close guards above, but I think that should be a separate change?